tests/lava/magma/runtime/test_runtime_service.py

```diff
@@ -1,14 +1,27 @@
+import logging
```

Comment:
Is there an unit test from the runtime level - i.e. if there is a single process model which is NeuroCore (simple for now) and you add that and then you call process.run() - If you need something in the compiler, you can mock it. But this test would demonstrate if the runtime -> NxSDKRuntimeService -> process_model flow is working.

If you want more clarity on the test, let's schedule a 30 min and we can quickly write up the unit test.

Reply:
I have the start of a unit test for "runtime -> NxSDKRuntimeService -> process_model" at bottom and in test_runtime_service.py. Since it was a task that seemed to require working channels between runtime and NxSDKRuntimeService I am working on it this week as part sprint work on runtime to runtimeservice channel communication.

In the unit test I was deadlocking on `cmd = self.service_to_process.recv()` in lava/magma/core/model/nc/model.py
and it seemed there were two problems. One I was not ACKing for each message. I believe I fixed this. Two I did not start up the runtime and thus NxSDKRuntimeService and NcProcModel were waiting indefinitely for a message to come from runtime.

On the second point I could use some help.

```
    # @unittest.skipUnless(run_loihi_tests, \
    #                      "runtime_to_runtime_service_to_process \
    #                      communication")
    @unittest.skip("runtime_to_runtime_service_to_process_to_loihi \
                   communication")
    def test_runtime_service_loihi_channels_start_run(self):
        node: Node = Node(HeadNode, [])
        exec: Executable = Executable()
        exec.node_configs.append(NodeConfig([node]))
        runtime: Runtime = Runtime(exec, ActorType.Mult
```